test/REGRESSIONS

```diff
@@ -568,7 +556,12 @@ Reviewed 2015-01-23
 
 all tests fail compilation with "$CHPL_HOME/modules/internal/ChapelDistribution.chpl:65: error: unresolved call '(bool)'"
 -------------------------------------------------------------------------------------------------------------------------
-(2015-01-19.. )
+(2015-01-19..2015-01-26)
```

Comment:
I see you're inheriting this, but it seems strange to me to put standalone dates below the '------' separator line.  In cases where all the tests under the line are subject to the same failure mode, I'd put the dates above the line since there isn't going to be any test-by-test differentiation.  Same with the case just below this.